src/SFA.DAS.ApprenticeFeedback.Domain/Models/Enums.cs

```diff
@@ -24,7 +24,8 @@ public enum FeedbackEligibilityStatus
 
         public enum ApprenticeshipStatus
         {
-            InProgress = 0,
+            Unknown = 0,
```

Comment:
@ChrisJWoodcock I've not looked at the feedback api so wouldnt know exactly what `ApprenticeshipStatus` is used for but could updating `InProgress` from 0 to 1 cause issues if this value is stored the db since the meaning has changed unless this is intentional?